### PR TITLE
delete comment in admin

### PR DESCRIPTION
**Pull request by lienht-1767**
==> T
==> Redmine: https://edu-redmine.sun-asterisk.vn/issues/8964
==> Attachment File:
Screenshot from 2019-03-26 14-43-28
Screenshot from 2019-03-26 14-43-48
Screenshot from 2019-03-26 14-45-01

### CODE REVIEW

**Comment by lienht-1767** [comment]
ready

File: app/controllers/admin/comments_controller.rb [hunk 1]
```diff
@@ -2,4 +2,14 @@ class Admin::CommentsController < Admin::BaseController
...
     @comments = Comment.page(params[:page]).includes(:user, :post).per Settings.per_page
   end
+
+  def destroy
+    @comment = Comment.find_by id: params[:id]
+    if @comment.destroy
+      flash[:success] = t ".deleted"
+      redirect_to request.referer
+    else
+      redirect_to admin_comments_url
```
Comment (huuphuc93) [inline app/controllers/admin/comments_controller.rb:13]: thêm flash error

**Comment by lienht-1767** [comment]
ready

**Comment by huuphuc93** [comment]
commented
gộp commit rồi ready lại

**Comment by lienht-1767** [comment]
ready

File: app/views/layouts/admin/_sidebar.html.erb [hunk 2]
```diff
@@ -9,9 +9,7 @@
     <%= link_to t(".posts"), "#" %>
   </li>
   <li>
-    <% if current_page?(admin_comments_url) %>
-      <%= link_to t(".comments"), admin_comments_url, class: "active" %>
-    <% end %>
+      <%= link_to t(".comments"), admin_comments_url%>
```
Comment (huuphuc93) [inline app/views/layouts/admin/_sidebar.html.erb]: indent

**Comment by lienht-1767** [comment]
ready

**Pull request merged**